src/main/kotlin/no/kartverket/matrikkel/bygning/matrikkel/adapters/ByggeaarDeriver.kt

```diff
@@ -0,0 +1,33 @@
+package no.kartverket.matrikkel.bygning.matrikkel.adapters
+
+import no.kartverket.matrikkel.bygning.matrikkelapi.MatrikkelBygningsstatusKode
+import no.kartverket.matrikkel.bygning.matrikkelapi.toLocalDate
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.Timestamp
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.bygning.koder.BygningsstatusKodeId
+import java.time.LocalDate
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.bygning.Bygning as MatrikkelBygning
+
+private val earliestDateForDerivingByggeaar = LocalDate.of(2009, 4, 25)
```

Comment:
For å tydeliggjøre at det er en konstant.

```suggestion
private val EARLIEST_DATE_FOR_DERIVING_BYGGEAAR = LocalDate.of(2009, 4, 25)
```